cmd/neo-exporter/config.go

```diff
@@ -33,7 +33,7 @@ const (
 
 func DefaultConfiguration(cfg *viper.Viper) {
 	cfg.SetDefault(prefix+delimiter+cfgNeoRPCEndpoint, "")
-	cfg.SetDefault(prefix+delimiter+cfgNeoRPCDialTimeout, 5*time.Second)
+	cfg.SetDefault(prefix+delimiter+cfgNeoRPCDialTimeout, time.Minute)
 
 	cfg.SetDefault(cfgMetricsEndpoint, ":16512")
 	cfg.SetDefault(cfgMetricsInterval, 15*time.Minute)
```

Comment:
This PR is for timeout changes, let's add one more commit and change this to `15*time.Second`.
The example YAML config contains `15s` and I doubt 15 minutes is a good interval between metric gathering for default value :sweat_smile: